### PR TITLE
refactor tutorials a little to make them more consistent with one another

### PR DESCRIPTION
**Pull request by aantn**
Also shorten a little where possible

### CODE REVIEW

**Comment by aantn** [comment]
@arikalon1 please review. We still need more work on the "Alert Enrichment" tutorial which currently isn't really a tutorial but is reference docs instead.

@Sheeproid tagging you too as I modified the tutorial you wrote a little. Feel free to comment.

File: docs/catalog/triggers/kubernetes.rst [hunk 1]
```diff
@@ -73,9 +73,16 @@ Additional triggers
 These triggers fire on very specific events:
 
 .. _on_kubernetes_warning_event:
+.. _on_kubernetes_warning_event_create:
+.. _on_kubernetes_warning_event_update:
+.. _on_kubernetes_warning_event_delete:
 
-* on_kubernetes_warning_event - when a Kubernetes event of level WARNING is created or modified
+* on_kubernetes_warning_event_create - when a Kubernetes event of level WARNING is created, modified, or deleted
```
Comment (arikalon1) [inline docs/catalog/triggers/kubernetes.rst:80]: i think this should be without the _create

**Comment by arikalon1** [comment]
nice.
Left one small comment

File: docs/catalog/triggers/kubernetes.rst (hunk 1 above)
Comment (aantn) [reply docs/catalog/triggers/kubernetes.rst]: Thanks, good catch

**Pull request merged**